torntools/scripts/content/education/ttEducation.js

```diff
@@ -1,15 +1,15 @@
 requireDatabase().then(() => {
-  console.log("TT - Education");
-  let observer = new MutationObserver(hideCompletedCategories);
-  observer.observe(doc.find(".content-wrapper.m-left20"), { childList: true });
-  hideCompletedCategories();
+	console.log("TT - Education");
+	let observer = new MutationObserver(hideCompletedCategories);
+	observer.observe(doc.find(".content-wrapper.m-left20"), { childList: true });
+	hideCompletedCategories();
 });
 
 function hideCompletedCategories() {
-  if (window.location.href.includes("step=main")) {
-    for (let category of doc.findAll(".education .ajax-act")) {
-      if (category.find(".bar-green-wrap-white-bg").style.width === "100%")
-        category.style.opacity = "0.2";
-    }
-  }
-};
+	if (window.location.href.includes("step=main")) {
+	for (let category of doc.findAll(".education .ajax-act")) {
+		if (category.find(".bar-green-wrap-white-bg").style.width === "100%")
+			category.style.opacity = "0.2";
```

Comment:
I'd personally add a CSS class and then set the opacity through the class (just to avoid in-line CSS) but it doesn't really matter here.

Reply:
Its just one line of CSS and adding a class makes it 3 lines in the CSS File and another line of code here to change its classes.

Reply:
Yea, that is true but just following the logic of 'what language is supposed to do what', I'd do it through CSS. But again, this isn't really important here like you said, it's only 1 line of code here.

Reply:
Yes.